gov-output/mainnet/targetsAndCalldata-7_9_2024-76864215.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ODGovernor_Address": "0xf704735CE81165261156b41D33AB18a08803B86F",
+  "arrayLength": "1",
+  "calldatas": [
+    "0x12f3352657455448000000000000000000000000000000000000000000000000000000007365636f6e646172795461785265636569766572000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000600000000000000000000000009c86c719aa29d426c50ee3baed40008d292b02cf000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000006f05b59d3b20000"
+  ],
+  "chainid": 42161,
+  "description": "add secondary tax receiver for WETH",
+  "descriptionHash": "0x42ba67fde5f25ce4c9185c3be6e859c5f2e4293c1321cfa063da8973c3b432c7",
+  "network": "mainnet",
+  "objectArray": [
+    {
+      "target": "0xc93F938A95488a03b976A15B20fAcFD52D087fB2",
+      "calldata": "0x12f3352657455448000000000000000000000000000000000000000000000000000000007365636f6e646172795461785265636569766572000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000600000000000000000000000009c86c719aa29d426c50ee3baed40008d292b02cf000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000006f05b59d3b20000"
+    }
+  ],
+  "proposalId": 76864215325714286555073332752783782717698752062872007843280380922333230264505,
```

Comment:
string proposal ID

Reply:
So it looks like forge will convert any string to integers if it contains only numbers.  the workaround was to do it in js after the output file is written.  It looks like this might be fixed in an upcomming update.  but untill then it's happening in the bash script.